src/modules/transaction/components/TransactionResultList/TransactionResultList.js

```diff
@@ -1,10 +1,23 @@
 import React from 'react';
 import { truncateAddress } from '@wallet/utils/account';
 import Icon from 'src/theme/Icon';
-import { getTxConfig } from '@search/utils';
+import { MODULE_COMMANDS_MAP } from 'src/modules/transaction/configuration/moduleCommand';
 import TokenAmount from '@token/fungible/components/tokenAmount';
 import styles from './TransactionResultList.css';
 
+const getTxConfig = (t, transactions) => {
+  const { params, fee } = transactions[0];
+
+  return {
+    icon:
+      transactions[0].moduleCommand === 'token:transfer'
```

Comment:
Can we not use `MODULE_COMMANDS_NAME_MAP.transfer` everywhere here instead of `'token:transfer'` ?